stolon/templates/sentinel-deploy.yaml

```diff
@@ -4,24 +4,21 @@ metadata:
   name: {{ template "stolon.sentinel.fullname" . }}
   labels:
     app: {{ template "stolon.sentinel.fullname" . }}
```

Comment:
what is the reasoning behind having half of the components with hardcoded app label and half with templated as before?